Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -69,14 +70,19 @@ public ExportDms(boolean exportImages) {
      *            Process object
      */
     @Override
-    public void startExport(Process process) throws IOException, DAOException {
+    public void startExport(Process process) throws DAOException, DataException, IOException {
+        boolean exported = process.isExported();
+        process.setExported(true);
+        ServiceManager.getProcessService().save(process);
         boolean exportSucessfull = startExport(process, (URI) null);
         if (exportSucessfull) {
-            process.setExported(true);
-        }
-        if (Objects.nonNull(process.getParent())) {
-            startExport(process.getParent());
+            if (Objects.nonNull(process.getParent())) {
+                startExport(process.getParent());
+            }
+        } else {
+            process.setExported(exported);
         }
+        ServiceManager.getProcessService().save(process);
```

Comment:
Is there a reason to save the process, when the process flagis not changed again? I think this save is only neccessary when export failed and exportflag changed back. It is indeed only neccessary if the flag ACTUALLY changed. This also counts for first save. So if the process is already exported, no save is needed at all